07-Animate-From-a-Variable-Point-with-from-and-fromTo-in-Greensock.md

```diff
@@ -0,0 +1,19 @@
+Use TweenMax.from
+    - [TweenMax.from docs](https://greensock.com/docs/v2/TweenMax/static.from())
+    - very similar to the TweenMax.to() used before, except the properties will be where the animation starts, and its current position will be where it ends.
+    ```
```

Comment:
Let's optimize to keep our markdown formatted. Code and Syntax Highlighting should not be indented. This applies for the rest of the lessons.